admin/class-help-center.php

```diff
@@ -0,0 +1,158 @@
+<?php
+/**
+ * @package WPSEO\Admin
+ */
+
+/**
+ * Class WPSEO_Help_Center
+ */
+class WPSEO_Help_Center {
+	/**
+	 * @var array Help_Center_Item list of helpcenter tabs to be shown in the helpcenter
+	 */
+	private $help_center_items = array();
+
+	/**
+	 * @var String $group_name
+	 */
+	private $group_name;
+
+	/**
+	 * @var WPSEO_Option_Tab $tab
+	 */
+	private $tab;
+
+	/**
+	 * WPSEO_Help_Center constructor.
+	 *
+	 * @param String           $group_name The name of the group of the tab the helpcenter is on.
+	 * @param WPSEO_Option_Tab $tab        The name of the tab the helpcenter is on.
+	 */
+	function __construct( $group_name, $tab ) {
+		$this->group_name = $group_name;
+		$this->tab        = $tab;
+
+		$kb_help_center_item = new WPSEO_Help_Center_Item(
+			'knowledge-base',
+			__( 'Knowledge base', 'wordpress-seo' ),
+			array(
+				'content'        => '<div class="wpseo-kb-search"></div>',
+				'view_arguments' => array( 'identifier' => $tab->get_name() ),
+			)
+		);
+		array_push( $this->help_center_items, $kb_help_center_item );
+
+		$popup_title = sprintf( __( 'Email support is a %s feature', 'wordpress-seo' ), 'Yoast SEO Premium' );
+		/* Translators: %1$s: expands to 'Yoast SEO Premium', %2$s: links to Yoast SEO Premium plugin page. */
```

Comment:
Please change `Translators` to `translators` so the translator comment is parsed correctly.